src/main/java/csci310group11/Implementation/CollageGenerator.java

```diff
@@ -1,17 +1,73 @@
 package csci310group11.Implementation;
 
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
 import java.awt.image.BufferedImage;
 import java.io.File;
+import java.util.ArrayList;
 import java.io.IOException;
-
 import javax.imageio.ImageIO;
 
 public class CollageGenerator {
+	private ArrayList<BufferedImage> images; //change to <BufferedImage> if necessary
+	private ArrayList<BufferedImage> borderedImages;
+	private BufferedImage collage;
+
+	public CollageGenerator(ArrayList<BufferedImage> images) {
+		this.images = new ArrayList<BufferedImage>();
+		this.borderedImages = new ArrayList<BufferedImage>();
+		this.collage = new BufferedImage();
```

Comment:
uhh i think it won't be able to initialize unless you initialize the size of the collage.